pandar_pointcloud_at128/launch/PandarSwift_points.launch

```diff
@@ -17,7 +17,7 @@
   <arg name="repeat_delay" default="0.0" />
   <arg name="rpm" default="600.0" />
   <arg name="start_angle" default="0.0" />
-  <arg name="publish_model" default="both_point_raw" />
+  <arg name="publish_model" default="points" />
```

Comment:
It looks like their usage of `points` vs. `point` is inconsistent between `pandar_pointcloud` and `pandar_pointcloud_at128`:
- `points`: https://github.com/seoulrobotics/HesaiLidar_Swift_ROS/blob/257ba6bf69cdb20dff7d55d99d2bab3996796dd2/pandar_pointcloud/src/conversions/driver.cc#L242 https://github.com/seoulrobotics/HesaiLidar_Swift_ROS/blob/257ba6bf69cdb20dff7d55d99d2bab3996796dd2/pandar_pointcloud/src/conversions/convert.cc#L237
- `point`: https://github.com/seoulrobotics/HesaiLidar_Swift_ROS/blob/257ba6bf69cdb20dff7d55d99d2bab3996796dd2/pandar_pointcloud_at128/src/conversions/convert.cc#L207 https://github.com/seoulrobotics/HesaiLidar_Swift_ROS/blob/257ba6bf69cdb20dff7d55d99d2bab3996796dd2/pandar_pointcloud_at128/src/conversions/driver.cc#L259

So this may need to be
```suggestion
  <arg name="publish_model" default="point" />
```

I'm not familiar with this codebase at all though. It's possible this launch file also uses `pandar_pointcloud` behind the scenes, though that doesn't seem likely.

Reply:
Great catch! Will change to point and check later with QA